### ISSUE TITLE
can i mock a  RPC service by using moco?

### ISSUE THREAD

**Issue by JennyHui** [dreamhead/moco#162]
all i got is the thrift file ~ i wanna know if moco can work this out or not?

**Comment by schrepfler** [comment]
No.
Additionally RPC is a generic term existing before thrift, this issue should be renamed as the title is misleading.

**Comment by dreamhead** [comment]
No, Moco supports HTTP and socket for now. You can write a thrift plugin for Moco to run thrift HTTP stub server

**Comment by darbean** [comment]
I forked ur moco and add the dubbo support.
